src/api/api.ts

```diff
@@ -0,0 +1,40 @@
+import { ICategory, IInputFormData } from "@/types/type"
+import { Json } from "database.types"
+import supabase from "supabase/supabase"
+
+interface RequestValue {
+  input_type: string
+  category: Json
+  amount: number
+  place: string
+  memo: string
+  date: string
+}
+
+const categoryToJson = (category : ICategory): Json => ({
+  id: category.id,
+  name: category.name,
+  value: category.value,
+})
+
+export const fetchData = async (input: IInputFormData) => {
```

Comment:
fetchData라는 이름이 무엇을 뜻하는지 잘 모르겠습니다. 명시적으로 바꿔 주심이 어떠실까요?
```suggestion
export const addData = async (input: IInputFormData) => {
```